cup_of_join.py

```diff
@@ -0,0 +1,18 @@
+#submitted by Keren Or Hadad 208025205
+
+#part 5.2
+
+def cup_of_join(*args, sep = '-'):
+    lst_return = []
+    for arg in args:
```

Comment:
Can we simplify this loop by using list comprehension?

Reply:
done